cmd/eventlistenersink/main.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	triggersclientset "github.com/tektoncd/triggers/pkg/client/clientset/versioned"
+	"github.com/tektoncd/triggers/pkg/sink"
+	kubeclientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+func main() {
+	log.Print("EventListener pod started")
+
+	// Get env variables
+	elName := os.Getenv("EL_NAME")
+	if elName == "" {
+		log.Fatal("EL_NAME env not found")
+	}
+	elNamespace := os.Getenv("EL_NAMESPACE")
+	if elNamespace == "" {
+		log.Fatal("EL_NAMESPACE env not found")
+	}
+	port := os.Getenv("PORT")
+	if port == "" {
+		log.Fatal("PORT env not found")
+	}
```

Comment:
iirc - I started with cmd line args in the experimental listener, and moved everything to env. At the time, building the cli args felt hackier and I think I ran into the common string formatting issues, but that was then. Whatever works best now is good with me.

Reply:
I personally don't have much of a preference between the two. Since args are inherently documented by the usage details, they're a little more verbose. So, I'll change these to args 👍